config_scheduler.go

```diff
@@ -27,8 +28,9 @@ func (c *SchedulerConfig) Get() error {
 func (c *SchedulerConfig) Set() error {
 	_, err := db.Upsert(ConfigCollection, byId(c.SectionId()), bson.M{
```

Comment:
why not do an update/replace rather than needing to set each field?

Reply:
Could you elaborate? We are doing an update on the fields here